internal/cli/root/builder.go

```diff
@@ -253,6 +253,11 @@ Use the --help flag with any command for more info on that command.`,
 	)
 
 	pluginCmd.RegisterCommands(rootCmd)
+	err := pluginCmd.RegisterFirstClassPluginCommands(rootCmd)
```

Comment:
Can you make `RegisterFirstClassPluginCommands` part of `RegisterCommands` so we can keep our original of having one line to register all plugins? (+ error handling ofc)

Reply:
done